arcgis-ios-sdk-samples/Content Display Logic/Controllers/Color Picker/ColorPicker.storyboard

```diff
@@ -0,0 +1,171 @@
+                                        <tableViewCellContentView key="contentView" opaque="NO" clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="center" ambiguous="YES" preservesSuperviewLayoutMargins="YES" insetsLayoutMarginsFromSafeArea="NO" tableViewCell="x0t-Ui-vFs" id="jOo-N7-oCq">
+                                            <rect key="frame" x="0.0" y="0.0" width="375" height="43.5"/>
+                                            <autoresizingMask key="autoresizingMask"/>
+                                            <subviews>
+                                                <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" verticalCompressionResistancePriority="1000" text="Hue" textAlignment="natural" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontForContentSizeCategory="YES" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="gDm-BC-Hek">
+                                                    <rect key="frame" x="16" y="12" width="31.5" height="21"/>
+                                                    <fontDescription key="fontDescription" style="UICTFontTextStyleBody"/>
+                                                    <nil key="textColor"/>
+                                                    <nil key="highlightedColor"/>
+                                                </label>
+                                                <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Label" textAlignment="right" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontForContentSizeCategory="YES" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="k5P-Xz-PeO">
+                                                    <rect key="frame" x="317" y="12" width="42" height="21"/>
+                                                    <fontDescription key="fontDescription" style="UICTFontTextStyleBody"/>
+                                                    <nil key="textColor"/>
+                                                    <nil key="highlightedColor"/>
+                                                </label>
+                                                <slider opaque="NO" contentMode="scaleToFill" verticalCompressionResistancePriority="999" contentHorizontalAlignment="center" contentVerticalAlignment="center" value="0.5" minValue="0.0" maxValue="1" translatesAutoresizingMaskIntoConstraints="NO" id="HLw-Qo-tVg">
+                                                    <rect key="frame" x="14" y="41" width="347" height="41"/>
+                                                    <constraints>
+                                                        <constraint firstAttribute="height" constant="40" id="WUO-uB-MMe"/>
```

Comment:
Sliders don't need their height constrained because they have an intrinsic content height.

Reply:
Ok. IB doesn't like sliders apparently:
![screen shot 2018-10-29 at 3 02 13 pm](https://user-images.githubusercontent.com/42748837/47683090-bea19e00-db8b-11e8-8eb0-cc43cc8de306.png)

All constraints are as expected and the live views are correct.

Reply:
I think IB doesn't like table view cells. It *should* resize the content view to fit its contents, but I couldn't get it to do that. Sigh.

Reply:
It works sometimes!